docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
```

Comment:
> some links can get quite long in markdown

Yes, that's why I said _"**were possible** try to keep the length..."_

> Referenced links seem to work better in those situations.

Can you give an example on how that would look like? Thanks!